content/en/logs/log_collection/csharp.md

```diff
@@ -24,76 +27,74 @@ further_reading:
   text: "Log Collection Troubleshooting Guide"
 ---
 
-To send your C# logs to Datadog, we recommend logging to a file and then tailing that file with your Datadog Agent. Here are setup examples for the `Serilog`, `NLog`, and `log4net` logging libraries
+To send your C# logs to Datadog, we recommend [logging to a file][1] and then [configuring the Datadog Agent][2] to tail the file. See example logging configurations for [Serilog][3], [NLog][4], and [log4net][5]. Datadog recommends setting up your logging library to output logs in JSON format to avoid the need for [custom parsing rules][6].  
```

Comment:
```suggestion
To send your C# logs to Datadog, [log to a file][1] and then [configuring the Datadog Agent][2] to tail the file. See the example logging configurations for [Serilog][3], [NLog][4], and [log4net][5]. Datadog recommends setting up your logging library to output logs in JSON format to avoid the need for [custom parsing rules][6].  
```